src/plasma/plasma_client.c

```diff
@@ -333,14 +342,18 @@ int64_t plasma_evict(plasma_connection *conn, int64_t num_bytes) {
 
 int plasma_subscribe(plasma_connection *conn) {
   int fd[2];
+  /* TODO: Just create 1 socket,
+   * bind it to port 0 to find a free port,
+   * and send the port number instead, and let the client connect.
+   */
```

Comment:
The communication over the port could have different performance though, right? Couldn't it be slower?